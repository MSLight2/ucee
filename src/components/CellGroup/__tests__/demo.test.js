import { demoTest } from '../../../tests/shared/demoTest';
import CellGroupDemo from '../demo';

demoTest('CellGroup', CellGroupDemo);