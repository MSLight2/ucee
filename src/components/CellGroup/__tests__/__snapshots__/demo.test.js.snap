// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CellGroup demo test 1`] = `
Array [
  <div
    className="ucee-cell-group"
  >
    <div
      className="ucee-cell"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <span>
          单元格
        </span>
      </div>
    </div>
    <div
      className="ucee-cell"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <span>
          单元格
        </span>
      </div>
      <div
        className="ucee-cell__value"
      >
        内容
      </div>
    </div>
  </div>,
  <div
    className="ucee-cell-group"
    style={
      Object {
        "marginTop": 10,
      }
    }
  >
    <div
      className="ucee-cell"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <span>
          单元格
        </span>
      </div>
    </div>
    <div
      className="ucee-cell"
      onClick={[Function]}
    >
      <div
        className="ucee-cell__title"
      >
        <span>
          单元格
        </span>
      </div>
      <div
        className="ucee-cell__value"
      >
        内容
      </div>
    </div>
  </div>,
]
`;
